client/src/interfaces/User.interface.ts

```diff
@@ -0,0 +1,23 @@
+interface IUser {
+  _id: string,
+  name: string,
+  allergens: Array<string>,
+  events: Array<any>,
+  uid: string,
+  about: string,
+  img: string
+}
+
+interface IUserContainerProps {
+  users: Array<IUser>
+}
```

Comment:
Can we turn this into a generic `IContainerProps` from now? Since the plan is to refactor the containers anyway.

```suggestion
interface IUserContainerProps<T> {
  users: Array<T>
}
```

Same comments as above about `Array<T>` vs `T[]`.